src/cryptography/hazmat/backends/__init__.py

```diff
@@ -16,7 +16,7 @@ def _available_backends():
     global _available_backends_list
 
     if _available_backends_list is None:
-        _available_backends_list = [
+        _entry_point_backends = [
```

Comment:
This shouldn't have an underscore, `_available_backends_list` does because it's the global name.